CONTRIBUTING.md

```diff
@@ -22,3 +22,71 @@ use GitHub pull requests for this purpose. Consult [GitHub Help] for more
 information on using pull requests.
 
 [GitHub Help]: https://help.github.com/articles/about-pull-requests/
+
+## Contributing labeled data
```

Comment:
From the form, we also have this info we could include:

***What types of datasets are useful?***
1. Large (More than 100k comments w/labels or more than 1M w/o labels)
2. If labeled: 
-----Good quantity/distribution of labels: at least 20k examples of labels being looked for, (e.g. toxic comments or rejected comments) 100k different labels occurring only once each is not useful.
-----High agreement with a professional reviewer. Have someone on your team or a professional re-review the labels for 100 examples.  For the data to be useful, the re-review should agree with the original labelling (e.g. >75% of the time).

Reply:
Also this:

Important note: Please do not include any personally identifiable  information (eg names, usernames, email addresses etc) in the datasets you share.

Reply:
I've added that data!